doc/DeveloperGuide.md

```diff
@@ -88,6 +88,44 @@ Use case ends.
 > 3a1. AddressBook shows an error message <br>
   Use case resumes at step 2
 
+
+#### Use case: Rename the tag 'friends' into 'buddies'
+
+Precondition : 'friends' tag exists in the address book
+**MSS**
+
+1. User requests to list all tags
+2. AddressBook shows a list of all tags
+3. User requests to rename the 'friends' tag
+4. AddressBook asks a new name for the tag
+5. User responds by typing 'buddies'
+6. AddressBook displays a message indicating the tag has been renamed
+Use case ends
+
+**Extensions**
+3a. User chooses a wrong tag to rename
+
+> 4.   AddressBook asks a new name for the tag
```

Comment:
Should be 3a1.

Reply:
should I still write 3a1. even though the step is exactly the same as step no.4 in MSS?

Reply:
Yes.. You can think of this extension as different scenario, so the numbering is important.

Reply:
Got it. Thx!